packages/components/pages/f-contact-preferences/src/components/ContactPreferences.vue

```diff
@@ -297,11 +341,15 @@ export default {
     }
 }
 
-.c-contactPreferences-btn {
+.c-contactPreferences-inheritWidthAboveNarrow {
     width: 100%;
 
     @include media('>narrow') {
         width: inherit;
     }
 }
+
+.c-contactPreferences-bottomMargin {
```

Comment:
If this is applied to all the `f-alert` components, I'd change the classname to be `c-contactPreferences-alert`. That way if the styling ever changes, the classname is a bit less tied to the bottomMargin styling.